src/components/Articles/CompanyProvisions.vue

```diff
@@ -0,0 +1,227 @@
+    <v-layout v-if="!isEditing">
+      <v-flex xs3>
+        <label><strong>Pre-existing Company Provisions</strong></label>
+        <v-chip v-if="hasProvisionsRemovedPropsChanged" x-small label color="#1669BB" text-color="white">
+          CHANGED
+        </v-chip>
+      </v-flex>
+      <v-flex xs7 class="info-text" v-if="provisionsRemoved">
+        The company has resolved that none of the Pre-existing Company Provisions are to apply to this company.
+      </v-flex>
+      <v-flex xs7 class="info-text" v-else>
+        This company has Pre-existing Company Provisions
+      </v-flex>
+      <v-flex mt-n2 xs2 class="align-right" v-if="!hasProvisionsRemovedPropsChanged">
+        <v-btn id="correct-company-provisions" text color="primary" @click="isEditing = true">
+          <v-icon small>mdi-pencil</v-icon>
+          <span>{{ editLabel }}</span>
+        </v-btn>
+      </v-flex>
+      <v-flex mt-n2 xs2 class="align-right" v-else>
+        <v-btn
+          id="undo-company-provisions"
+          text
+          color="primary"
+          class="undo-company-provisions"
+          @click="resetCompanyProvisions"
+        >
+          <v-icon small>mdi-undo</v-icon>
+          <span>Undo</span>
+        </v-btn>
+
+        <!-- More Actions Menu -->
+        <span class="more-actions">
+          <v-menu offset-y>
+            <template v-slot:activator="{ on }">
+              <v-btn text small color="primary" class="more-actions-btn" v-on="on">
+                <v-icon>mdi-menu-down</v-icon>
+              </v-btn>
+            </template>
+            <v-list>
+              <v-list-item class="actions-dropdown_item" @click="isEditing = true">
+                <v-list-item-subtitle>
+                  <v-icon small>mdi-pencil</v-icon>
+                  <span class="ml-1">{{ editLabel }}</span>
+                </v-list-item-subtitle>
+              </v-list-item>
+            </v-list>
+          </v-menu>
+        </span>
+      </v-flex>
+    </v-layout>
+    <v-layout v-else>
+      <v-flex xs3>
+        <label><strong>Pre-existing Company Provisions</strong></label>
+      </v-flex>
+      <v-flex xs9>
+        <v-layout>
+          <v-flex>
+            <p class="instructions-paragraph">
+              Complete this item only if the company has resolved that none of the Pre-existing Company Provisions
+              are to apply to this company (refer to Part 17 and Table 3 of the Regulation under the Business
+              Corporations Act).
+            </p>
+            <v-layout>
+                <v-checkbox
+                  id="cp-checkbox"
+                  class="mt-1"
+                  :color="isInvalid ? checkBoxColor : ''"
+                  v-model="draftProvisionsRemoved"
+                />
+                <span class="checkbox-label" :class="{ 'invalid': isInvalid }">
+                  The company has resolved that none of the Pre-existing Company Provisions are to apply to this company
```

Comment:
Add period at end?

(Does the ux spec explicitly have no end period???)

Reply:
We should add the period. Will come in the next commit. Thanks.

Reply:
Done.